trunk/source/CACSL2BoogieTranslator/src/de/uni_freiburg/informatik/ultimate/cdt/translation/implementation/result/ContractResult.java

```diff
@@ -48,7 +48,8 @@ public class ContractResult extends Result {
 	/**
 	 * Constructor.
 	 *
-	 * @param specs a specification array.
+	 * @param specs
+	 *            a specification array.

```

Comment:
One thing I noticed here: We always use tabs for indentation, but the parameter description is indented using spaces. This was not adapted recently, but is already in the coding conventions for a long time, therefore changing it, might cause a lot of changes...

Reply:
Hm, the amount of changes does not really concern me (should it?) -- and it looks nicer, so why not?